orchagent/orchdaemon.cpp

```diff
@@ -1222,11 +1223,20 @@ bool DpuOrchDaemon::init()
 {
     SWSS_LOG_NOTICE("DpuOrchDaemon init...");
     OrchDaemon::init();
+
+    // Enable Dash ZMQ with CONFIG_DB flag
+    ZmqServer *dash_zmq_server = nullptr;
+    if (get_feature_status("orch_dash_zmq_enabled", true))
```

Comment:
@liuh-80 a minor comment. This feature name can be confusing as this handles zmq enablement only for the northbound side (gnmi etc) from orchagent perspective. The orchagent southbound interfaces w/ syncd is controlled separately and is always enabled for DPUs. Can we come up with a different name for this feature and also change the log below to be more specific (i.e this is zmq for the northbound interface).

Reply:
Fixed